debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/legacy/BinlogReaderMetrics.java

```diff
@@ -152,8 +152,14 @@ public void setMilliSecondsBehindSource(long value) {
     }
 
     @Override
+    @Deprecated
```

Comment:
Could you add a JavaDoc tag referring to the replacement, e.g.:

    @deprecated("Superseded by the 'Captured Tables' metric. Scheduled for removal in a future release")